scripts/build/TestPlatform.Dependencies.props

```diff
@@ -34,6 +34,7 @@
     <MoqVersion>4.7.63</MoqVersion>
     <TestPlatformExternalsVersion>16.10.0-preview-2-31108-246</TestPlatformExternalsVersion>
     <CodeCoverageExternalsVersion>16.10.0-beta.21175.5</CodeCoverageExternalsVersion>
+    <InteropExternalsVersion>17.0.0-preview-1-30928-1111</InteropExternalsVersion>
```

Comment:
New version `17.0.0-preview-1-30928-1112` just uploaded recently, it is better to use that, it contains some new fixes.

Reply:
Version `17.0.0-preview-1-30928-1112` fails on package restore due to a dependency on `Microsoft.VisualStudio.Imaging.Interop.14.0.DesignTime` so we have to stick to this one for now.